backend/restapi/urls.py

```diff
@@ -42,4 +43,9 @@
         name="get_files_by_mission_id",
     ),
     path("auth/", include("dj_rest_auth.urls")),
+    path(
+        "missions/<str:file_path>/files/topics",
```

Comment:
```suggestion
        "topics/<path:file_path>",
```
You need to use `path` here, because `str` is actually not a python string but a converter by django, which doesn't include `/`  but a file path includes `/ ` (at least on linux, so also at deployment). \
See more [here](https://docs.djangoproject.com/en/5.1/topics/http/urls/#path-converters) about available path converters.\
Also I think you can shorten the url path to just `topics/<path:file_path>`